src/main/java/com/auth0/client/mgmt/UsersEntity.java

```diff
@@ -204,7 +204,7 @@
     }
 
     /**
-     * Request all the Guardian Enrollments for a given User.
+     * Retreive the first confirmed enrollment, or a pending enrollment if none are confirmed.
```

Comment:
> Have we confirmed that it will return "a pending enrollment if none are confirmed"?

Just to close the loop, this is correct, and the API docs page should be updated as well.